packages/wrangler/src/dev/start-server.ts

```diff
@@ -99,12 +104,31 @@ export async function startDevServer(
 			inspector: {
 				port: props.inspectorPort,
 			},
-			urlOverrides: {
+			origin: {
 				secure: props.upstreamProtocol === "https",
 				hostname: props.localUpstream,
 			},
 			liveReload: props.liveReload,
 			remote: !props.local,
+			auth: async () => {
+				let accountId = props.accountId;
+				if (accountId === undefined) {
+					const accountChoices = await getAccountChoices();
+					if (accountChoices.length === 1) {
+						saveAccountToCache({
+							id: accountChoices[0].id,
+							name: accountChoices[0].name,
+						});
+						accountId = accountChoices[0].id;
+					} else {
+						throw logger.error(
+							"In a non-interactive environment, it is mandatory to specify an account ID, either by assigning its value to CLOUDFLARE_ACCOUNT_ID, or as `account_id` in your `wrangler.toml` file."
```

Comment:
```suggestion
							"Missing account ID.
In a non-interactive environment, it is mandatory to specify an account ID. Please: 
❯❯ specify it in your wrangler.toml file: `account_id = <your_account_id>, or
❯❯ assign its value to the CLOUDFLARE_ACCOUNT_ID environment variable: CLOUDFLARE_ACCOUNT_ID = <your_account_id>
```

Reply:
or smth along those lines...

Reply:
This error message is likely from existing code. More interesting, why are we throwing logger.error (which returns void)?